packages/webdriverio/src/commands/browser/keys.ts

```diff
@@ -54,7 +54,15 @@ export async function keys (
      * W3C way of handle it key actions
      */
     const keyAction = this.action('key')
-    keySequence.forEach((value) => keyAction.down(value))
+    keySequence.forEach((value) => {
+        if (!this.isIOS && value === ','){
+            /**
+             * comma can't be typed within sequence without extra pause here more than 200ms
+             */
+            keyAction.pause(225)
```

Comment:
Can we make this a `const` and move the variable declaration to the top of the file?